src/erlcloud_ddb2.erl

```diff
@@ -3718,7 +3784,7 @@ untag_resource(ResourceArn, TagKeys) ->
                   -> untag_resource_return().
 untag_resource(ResourceArn, TagKeys, Config) ->
     erlcloud_ddb_impl:request(
-      Config,
+      Config, false,
```

Comment:
This should probably call the /3 (false by default) version of the implementation, if you decide to go that route.